### ISSUE TITLE
Color of the liquid

### ISSUE THREAD

**Issue by ozanarmagan** [sagieppel/Detecting-and-segmenting-and-classifying-materials-inside-vessels-in-images-using-convolutional-net#4]
Hello,
First of all this is a great project! Secondly, is it possible to detect color of the liquid inside vessels?

**Comment by sagieppel** [comment]
No, but can just average the RGB value of the area and check the ration of
red to green and blue.
Alternatively, look at this
https://github.com/sagieppel/Predicting-Material-properties-of-objects-and-liquids-inside-transparent-vessels-from-image
